src/components/GoogleMap.js

```diff
@@ -0,0 +1,43 @@
+/**
```

Comment:
I think we can get rid of Google Maps entirely and just use Leaflet.

Reply:
Confirmed in the meeting today we will allow a single setting to change between Gmaps and OSM. OSM should be default and switchable to gmaps. Marking as resolved at this time.